yarn-project/acir-simulator/src/avm/avm_state_manager.ts

```diff
@@ -42,4 +43,23 @@ export class AvmStateManager {
     const journal = AvmJournal.branchParent(parent.journal);
     return new AvmStateManager(parent.blockHeader, journal);
   }
+
+  /**
+   * Passes storage call to the journal
+   * @param contractAddress -
+   * @param slot -
+   * @param value -
+   */
+  public store(contractAddress: AztecAddress, slot: Fr, value: Fr): void {
```

Comment:
I was going to ask you for tests for this, but given that you remove this class in a later PR feel free to resolve the comment.